docs/components/arm/lite6.md

```diff
@@ -102,6 +102,41 @@ See [the frame system service](/services/frame-system/) for more information on
 
 {{< readfile "/static/include/components/test-control/arm-control.md" >}}
 
+### DoCommand
+
+Execute the model-specific commands `set_speed` and `set_acceleration` that are not otherwise defined by the component API.
+
+{{< tabs >}}
+{{% tab name="Go" %}}
```

Comment:
@nfranczak want to confirm-- this is only implemented in the Go SDK? from what I could tell in the linked pr to the ticket there were no python or flutter changes

Reply:
Correct this is only implemented in the Go SDK for only the xArm models.

Reply:
I don't think you two are talking about the same thing. I think Sierra wants to know if we can call the model-specific commands from Python and flutter code as well. Like this for python for example:

```python
command = { "set_speed": 45.0, "set_acceleration": 10.0}
result = await my_arm.do_command(command)
```

I'm reasonably certain that should be possible given that the language a model is implemented in broadly doesn't matter. You can write code in _any_ language when you're using it. Whether a method is implemented in a specific SDK only matters for the method (so it matters that `DoCommand` exists in Flutter and Go for the arm component) - the command passed to the DoCommand method will just be passed on and eventually be executed by the go model code. But that does not mean we can't use python code to invoke it.

@nfranczak please confirm that this isn't some weird edge case with how DoCommand works?

Reply:
> I think Sierra wants to know if we can call the model-specific commands from Python and flutter code as well.

I understand, thank you for the clarification.
I just tested out the doCommand on hardware using the Python SDK and it seems to work. I am guessing that it will also work through flutter then.

Reply:
Thank you for testing 😊